calebwagner/sink-repair
# created service form for html

I created the html format for the created service requests then exported it to the main html module called SinkRepair.js.

Steps to check code:
1. git fetch --all
2. git checkout cjw-service-requests-chapter
3. code .
4. check ServiceForm.js module for html in ServiceForm function
5. make sure it was imported to SinkRepair.js which will render the html to the browser
